mandatory/3-function-output.js

```diff
@@ -1,16 +1,22 @@
 // Add comments to explain what this function does. You're meant to use Google!
+// This function adds a random number between 0 - 10, but excludes 10.
 function getNumber() {
   return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
+// This function is used to join 2 or more strings together, in this case, w1 and w2.
 function s(w1, w2) {
   return w1.concat(w2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
   // Write the body of this function to concatenate three words together.
   // Look at the test case below to understand what this function is expected to return.
+  var sentence = firstWord +" " +  secondWord +" " + thirdWord;
+  // var sentence = `${firstWord} ${secondWord} ${thirdWord}`
+  console.log(sentence);
```

Comment:
It's normally best to remove console.logs() before you submit your coursework if you don't need them 

https://syllabus.codeyourfuture.io/guides/marking-guide#debug-logging